web/src/App.jsx

```diff
@@ -18,6 +19,7 @@ function App() {
 				<Route path="/subscribe" element={<Subscribe />} />
 				<Route path="/subscribe/confirmation" element={<ConfirmationPage />} />
 				<Route path="/subscribe/error" element={<ErrorOnSubscribe />} />
+				<Route path="/upload/FileUploading" element={<FileUploading />} />
```

Comment:
Could we simplify this to `<Route path="/upload" element={<Upload />} />`? This would follow best practices by:

- Using lowercase URI 
- Avoiding redundant path segments

Reply:
Done

Reply:
Done